README.md

```diff
@@ -1,2 +1,47 @@
 # csw-infra
 Cloud Security Watch - Infrastructure
+
+## Instructions
+
+The below does not cover how you supply your AWS credentials to terraform. 
+
+This script allows you to deploy the same terraform configuration to multiple environments in separate accounts and VPCs. 
+
+This means that in our AWS test account we can defined a VPC for each developer containing their own public and private subnets and a small 
+RDS instance. 
+
+Similarly we can use the same script to provision multiple separate 
+test environments uat, qa... 
+
+Because the tfstate lives in a separate folder you can have multiple independent tfstates rather than one terraform apply destroying the resources created by another. 
```

Comment:
does this work even in case we use S3 to save the tfstate? Is the directory structure being preserved in S3?

Reply:
Yes - If we refactor to use modules like the cyber-security-infrastructure repo.